dump/helper.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+# Generate a Template which will be returned by Executor Classes
+def display_template(dbs):
+    template = {}
+    for db in dbs:
+        template[db] = {}
+        template[db]['keys'] = []
+        template[db]['tables_not_found'] = []     
+    return template
+
+def verbose_print(str):
```

Comment:
Instead of using an environment variable, can't we print verbose based on an option in the command? (like -v)

Reply:
When -v is passed to the top level CLI, this environ variable is set. This logic is in other PR's. Now, if we choose to remove environ variable, we could pass this an argument. But this would require match-engine, etc to have the knowledge of what was passed to the CLI. Hence, i believe ENVIRON variable is a better option here